docs/markdown/Getting Started/getting-started/manual-installation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Manual and advanced installation"
```

Comment:
Nice, I like that.

Did you create the page in Readme? Sounds like it has to be done there for new stuff.

Reply:
Pinged you in the Slack thread.

Reply:
Eeeew.

I presume you'll push the docs once I merge. I'll add the page in a hidden state to 2.12 and 2.13 in the meantime.

Reply:
> you'll push the docs

I wasn't planning on it. Feel free to though